.circleci/test-deploy.yml

```diff
@@ -222,17 +228,16 @@ jobs:
             grep -q "runner.addPlugin(FailOnWarningsPlugin())" console.log
             rm console.log
           shell: bash
-            # Running test in parallel is currently unsupported
-            #      - matlab/run-tests:
-            #          use-parallel: true
-            #          source-folder: src
-            #      - run:
-            #          name: Verify tests ran in parallel
-            #          command: |
-            #            set -e
-            #            grep -q "parallel pool" console.log
-            #            rm console.log
-            #          shell: bash
+      # - matlab/run-tests:
+      #     use-parallel: true
+      #     source-folder: src
+      # - run:
+      #     name: Verify tests ran in parallel
+      #     command: |
+      #       set -e
+      #       grep -q "parallel pool" console.log
+      #       rm console.log
+      #     shell: bash
```

Comment:
What's going on again with parallel support in run-tests? I remember Azure was facing the same issue, but I can't remember what the root issue actually is.

Reply:
We weren't able to support it with our v0 licensing mechanism. In the beta we were able to support it for a while, but with some recent changes it's broken again.

See:
https://github.com/mathworks/matlab-circleci-orb/issues/60

It's been reported to the corresponding team internally and is being worked on.